test/views/VatReturnDetailsViewSpec.scala

```diff
@@ -215,6 +217,93 @@ class VatReturnDetailsViewSpec extends ViewBaseSpec {
     }
   }
 
+  "Rendering the vat return details page from the returns route with flat rate scheme for an agent" should {
+
+    lazy val view = views.html.returns.vatReturnDetails(vatReturnViewModel)(fakeRequestWithClientsVRN, messages, mockConfig, Lang.forCode("en"), agentUser)
+
+    lazy implicit val document: Document = Jsoup.parse(view.body)
+
+    "not render breadcrumbs which" in {
+      an[TestFailedException] should be thrownBy element(Selectors.btaBreadcrumb)
+      an[TestFailedException] should be thrownBy element(Selectors.btaBreadcrumbLink)
+      an[TestFailedException] should be thrownBy element(Selectors.vatBreadcrumb)
+      an[TestFailedException] should be thrownBy element(Selectors.vatBreadcrumbLink)
+      an[TestFailedException] should be thrownBy element(Selectors.previousPageBreadcrumb)
+      an[TestFailedException] should be thrownBy element(Selectors.previousPageBreadcrumbLink)
+      an[TestFailedException] should be thrownBy element(Selectors.currentPage)
+    }
+
+    "have the correct subheading" in {
+      elementText(Selectors.subHeading) shouldBe "Return total: £1,000 You owe HMRC: £1,000"
```

Comment:
This content doesn't need to be tested again either

Reply:
removed the content